docs/content/latest/api/ysql/the-sql-language/statements/cmd_call.md

```diff
@@ -0,0 +1,174 @@
+---
+title: CALL statement [YSQL]
+headerTitle: CALL
+linkTitle: CALL
+description: Use the CALL statement to execute a stored procedure.
+menu:
+  latest:
+    identifier: cmd_call
+    parent: statements
+isTocNested: true
+showAsideToc: true
+---
+
+## Synopsis
+
+Use the `CALL` statement to execute a stored procedure.
+
+## Syntax
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#grammar" class="nav-link active" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
+      <i class="fas fa-file-alt" aria-hidden="true"></i>
+      Grammar
+    </a>
+  </li>
+  <li>
+    <a href="#diagram" class="nav-link" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
+      <i class="fas fa-project-diagram" aria-hidden="true"></i>
+      Diagram
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="grammar" class="tab-pane fade show active" role="tabpanel" aria-labelledby="grammar-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/call_procedure,procedure_argument,argument_name.grammar.md" /%}}
+  </div>
+  <div id="diagram" class="tab-pane fade" role="tabpanel" aria-labelledby="diagram-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/call_procedure,procedure_argument,argument_name.diagram.md" /%}}
+  </div>
+</div>
+**Note:** The syntax and semantics of the `procedure_argument` (for example how to use the named parameter invocation style to avoid providing actual arguments for defaulted parameters) is the same for invoking a user-defined`FUNCTION`. A function cannot be invoked with the `CALL` statement. Rather, it's invoked as (part of) an expression in DML statements like `SELECT`.
+
+
+## Semantics
+
+`CALL` executes a stored procedure. If the procedure has any output parameters, then a result row will be returned, containing the values of those parameters.
+
+The caller must have _both_ the `USAGE` privilege on the schema in which the to-be-called procedure exists _and_ the  `EXECUTE` privilege on it. If the caller lacks the required `USAGE` privilege, then this causes error 42501 (_"permission denied for schema %"_). If the caller has the required `USAGE` privilege but lacks the required `EXECUTE` privilege, then this causes error 42501 (_"permission denied for procedure %"_).
+
+## Notes
+
+If `CALL` is executed in a transaction block, then it cannot execute transaction control statements. The attempt causes the runtime error 2D000 (_"invalid transaction termination"_). Transaction control statements are  allowed when `CALL` is invoked with `AUTOCOMMIT` set to `on`—in which case the procedure executes in its own transaction.
```

Comment:
Error text and code text are not italicized. Use Markdown backticks. For formatting of text, see https://developers.google.com/style/text-formatting/#code-font.